src/linux/linux_io.c

```diff
@@ -625,6 +705,26 @@ static int run_io_all_interfaces(struct eventloop *loop, const struct cmdline_co
 		goto start_jetws_server_failed;
 	}
 
+	int uds_fd = create_server_unix_domain_socket(UDS_PATH);
+	if (uds_fd < 0) {
+		return -1;
+	}
+
+	struct jet_server jet_uds_server = {
+		.ev = {
+			.read_function = accept_jet,
+			.write_function = NULL,
+			.error_function = accept_jet_error,
+			.loop = loop,
+			.sock = uds_fd}};
+
+	ret = start_server(&jet_uds_server.ev);
+	if (ret < 0) {
+		close(uds_fd);
+		return -1;
+	}
+
+
```

Comment:
Would be nice to factor out that stuff into a function to avoid code duplication.

Reply:
done